### PR TITLE
luacheck: Fix some of the "accessing undefined variable" bugs

### PR DESCRIPTION
**Pull request by tpmanley**
Ran `luacheck --std lua53 ./ | grep "accessing undefined variable"` against the repo and found some bugs. This PR doesn't fix all of the issues found, but the remainder will take a bit more time and I didn't want to wait to get these fixes in.

### CODE REVIEW

**Review by greens — APPROVED** [review]

**Review by dljsjr — APPROVED** [review]

File: drivers/SmartThings/zwave-switch/src/aeon-smart-strip/init.lua [hunk 1]
```diff
@@ -109,9 +109,6 @@ local aeon_smart_strip = {
       [Meter.REPORT] = meter_report_handler
     }
   },
-  lifecycle_handlers = {
-    init = device_init
```
Comment (tpmanley) [inline drivers/SmartThings/zwave-switch/src/aeon-smart-strip/init.lua:113]: I do wonder if this was expected to do something important

Comment (cjswedes) [reply drivers/SmartThings/zwave-switch/src/aeon-smart-strip/init.lua:113]: ¯\_(ツ)_/¯ It hasn't been doing anything, so it must not be that important. There is usually not much that is needed on zwave device_init anyway.

**Review by cjswedes — APPROVED** [review]

**Review by sjl2178 — APPROVED** [review]

File: drivers/SmartThings/zwave-switch/src/aeon-smart-strip/init.lua (hunk 1 above)
Comment (greens) [reply drivers/SmartThings/zwave-switch/src/aeon-smart-strip/init.lua:113]: nothing in the entire history of the file that I can see

**Review by greens — APPROVED** [review]

**Comment by tpmanley** [comment]
Driver tests failing are a known issue but unrelated to this PR so I'm going to merge

**Pull request merged**